api/scripts/dump_data.py

```diff
@@ -19,11 +14,11 @@ def default(self, z):
         return super().default(z)
 
 
-engine = create_engine(SETTINGS.database_url, connect_args={"check_same_thread": False})
-meta = MetaData()
-meta.reflect(bind=engine)  # http://docs.sqlalchemy.org/en/rel_0_9/core/reflection.html
-result = {}
-for table in meta.sorted_tables:
-    result[table.name] = [dict(row) for row in engine.execute(table.select())]
-with open(outfile_path, "w") as f:
-    f.write(json.dumps(result, cls=BytesEncoder, indent=4))
+def dump():
+    """Dumps all entries from the database into 'test_data.json'"""
+    result = {}
+    for table in meta.sorted_tables:
+        if table.name != "alembic_version":
+            result[table.name] = [dict(row) for row in engine.execute(table.select())]
+    with open(file_path, "w") as f:
+        f.write(json.dumps(result, cls=BytesEncoder, indent=4))
```

Comment:
If we add these few lines, the script also remains executable on its own
```suggestion
        f.write(json.dumps(result, cls=BytesEncoder, indent=4))
      
if __name__ == "__main__":
    dump()
```